dnf-behave-tests/features/plugins-core/debuginfo-install.feature

```diff
@@ -33,10 +35,235 @@ Scenario: reports an error for a package without debuginfo
+  And Transaction is following
+      | Action        | Package                                   |
+      | install       | baz-debuginfo-0:2.0-1.x86_64              |
+      | install       | baz-debugsource-0:2.0-1.x86_64            |
+
+@not.with_os=rhel__eq__8
+Scenario: installs debuginfo when a -debuginfo package is provided
+ When I execute dnf with args "debuginfo-install foo-debuginfo"
+ Then the exit code is 0
+  And Transaction is following
+      | Action        | Package                                   |
+      | install       | foo-debuginfo-0:2.0-1.x86_64              |
+      | install       | foo-debugsource-0:2.0-1.x86_64            |
+
+@not.with_os=rhel__eq__8
+@bz1586059 @bz1629412
+Scenario: can't find the version of debuginfo for an installed package
+Given I successfully execute dnf with args "install bar-2.0"
+ When I execute dnf with args "debuginfo-install bar"
+ Then the exit code is 0
+  And stdout is
+      """
+      <REPOSYNC>
+      Could not find debuginfo package for the following installed packages: bar-2.0-1.x86_64
+      Could not find debugsource package for the following installed packages: bar-2.0-1.x86_64
+      Dependencies resolved.
+      Nothing to do.
+      Complete!
+      """
+
+@not.with_os=rhel__eq__8
+Scenario: can't find the requested version of debuginfo (multilib package, architectures can be mixed up)
+ When I execute dnf with args "debuginfo-install baz-1.0"
+ Then the exit code is 0
+  And stdout is
+      """
+      <REPOSYNC>
+      Could not find debuginfo package for the following available packages: baz-1.0-1
+      Could not find debugsource package for the following available packages: baz-1.0-1
+      Dependencies resolved.
+      Nothing to do.
+      Complete!
+      """
+
+@not.with_os=rhel__eq__8
+Scenario: installs debuginfo with the same (secondary) architecture as the installed package
+Given I successfully execute dnf with args "install baz.i686"
+ When I execute dnf with args "debuginfo-install baz"
+ Then the exit code is 0
+  And Transaction is following
+      | Action        | Package                                   |
+      | install       | baz-debuginfo-0:2.0-1.i686                |
+      | install       | baz-debugsource-0:2.0-1.i686              |
+
+@not.with_os=rhel__eq__8
+Scenario: installs debuginfo for multiple installed architectures
+Given I successfully execute dnf with args "install baz.i686 baz.x86_64"
+ When I execute dnf with args "debuginfo-install baz"
+ Then the exit code is 0
+  And Transaction is following
+      | Action        | Package                                   |
+      | install       | baz-debuginfo-0:2.0-1.i686                |
+      | install       | baz-debuginfo-0:2.0-1.x86_64              |
+      | install       | baz-debugsource-0:2.0-1.i686              |
+      | install       | baz-debugsource-0:2.0-1.x86_64            |
+
+@not.with_os=rhel__eq__8
+Scenario: installs debuginfo for multiple installed architectures
```

Comment:
Did you mean "versions" instead of "architectures"?

Reply:
Copy-paste remnant, thanks.